exporters/otlp/otlplog/otlploggrpc/client.go

```diff
@@ -9,6 +9,8 @@ import (
 	"fmt"
 	"time"
 
+	collogpb "go.opentelemetry.io/proto/otlp/collector/logs/v1"
```

Comment:
Temporarily holding a skeptical attitude: 

Is our combination based on `go.opentelemetry.io` or the current package (`go.opentelemetry.io/contrib`)